app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -398,6 +398,10 @@
   <dimen name="section_title_text_view_layout_margin_top">32dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">120dp</dimen>
 
+  <!--  PromotedStoryList-->
+  <dimen name="promoted_story_list_layout_margin_end">120dp</dimen>
+  <dimen name="promoted_story_list_layout_margin_start">120dp</dimen>
```

Comment:
`home_tablet_port_outer_margin ` should get removed if not getting used anywhere else.

Reply:
Ditto for `home_tablet_land_outer_margin`

Reply:
I already removed home_tablet_port_outer_margin and home_tablet_land_outer_margin from dimens file.... and it also doesn't exist in values-sw600dp-port/dimens , can u please tell me which file are u talking about..